cmd/create/cluster/cmd.go

```diff
@@ -201,7 +201,8 @@ func init() {
 		"Prefix to use for all IAM roles used by the operators needed in the OpenShift installer. "+
 			"Leave empty to use an auto-generated one.",
 	)
-
+	flags.MarkDeprecated("operator-roles-prefix", "instead run 'rosa create operator-roles' "+
```

Comment:
I'm really debating with myself on removing this flag entirely. Supporting two disparate flows in the same command is a pain. What do you think? Anything that you can think will break if we remove it entirely?

Reply:
We have support in CS to handle the operator roles coming in so I don't see any issues removing them